package.json

```diff
@@ -9,7 +9,8 @@
     "@salesforce/core": "^8.0.1",
     "@salesforce/kit": "^3.1.6",
     "@salesforce/sf-plugins-core": "^11.1.2",
-    "change-case": "^5.4.4"
+    "change-case": "^5.4.4",
+    "gulp": "^5.0.0"
```

Comment:
can you remove this dependency?

Reply:
@WillieRuemmele  Done. Not sure how that got in there, but ...